packages/vkui/src/components/Accordion/Accordion.tsx

```diff
@@ -1,8 +1,10 @@
-import React, { HTMLProps } from 'react';
+import React, { DetailsHTMLAttributes } from 'react';
```

Comment:
```suggestion
import * as React from 'react';
```

Для консистентности.